src/report.js

```diff
@@ -44,6 +45,10 @@ class Report {
       memoryLimitInMB
     } = this.context;
 
+    const pluginConfigss = plugins.map(pluginFn => {
```

Comment:
We will need to chat about this to make sure the info that a plugin passes up to `iopipe` is what we want long-term.